### PR TITLE
Revert #13908

### PR DESCRIPTION
**Pull request by krzyzacy**
let's see if this is the culprit first

also I'm giving up on prow/kube/client... let's rewrite plank...

/assign @fejta @Katharine @stevekuznetsov

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
What was this for?

**Comment by alvaroaleman** [comment]
> also I'm giving up on prow/kube/client... let's rewrite plank...

Please!

**Comment by krzyzacy** [comment]
@stevekuznetsov https://kubernetes.slack.com/archives/C7J9RP96G/p1565912286145300?thread_ts=1565905024.144300&cid=C7J9RP96G

for whatever reason plank sync loop seems took forever (the patch call?!?!)

we rolled back the deployment already

**Comment by krzyzacy** [comment]
/hold

actually... I probably can add some logs and timing

**Comment by krzyzacy** [comment]
@fejta maybe let's hold off bump until Monday, I'd like to investigate a bit more here, as `kubectl patch prowjobs` locally is pretty fast... I suspect the request stuck in retries for some reason?

File: prow/kube/client.go [hunk 1]
```diff
@@ -527,21 +527,6 @@ func (c *Client) ReplaceProwJob(name string, job prowapi.ProwJob) (prowapi.ProwJ
 	return retJob, err
 }
 
-// PatchProwJob will patch name with job in the client's specified namespace with a merge patch.
-//
-// Analogous to kubectl patch prowjobs/NAME --namespace=client.namespace --patch=PATCH
-func (c *Client) PatchProwJob(name string, patch []byte) (prowapi.ProwJob, error) {
-	c.log("PatchProwJob", name, string(patch))
```
Comment (krzyzacy) [inline prow/kube/client.go]: w.t.... I didn't even get this log?!

Comment (krzyzacy) [reply prow/kube/client.go]:
```
{"component":"plank","file":"prow/plank/controller.go:587","func":"k8s.io/test-infra/prow/plank.(*Controller).patchProwjob","level":"info","msg":"Created merge patch: {\"status\":{\"build_id\":\"1162448521480114176\",\"description\":\"Job triggered.\",\"pod_name\":\"4e9306fa-c05d-11e9-b87f-0a580a2c022d\",\"state\":\"pending\",\"url\":\"https://prow.gflocks.com/view/gcs/oss-prow/pr-logs/pull/GoogleCloudPlatform_oss-test-infra/51/pull-prow-config-validate/1162448521480114176\"}}","time":"2019-08-16T19:42:31Z"}
{"component":"plank","error":"errors syncing: [http: read on closed response body], errors reporting: []","file":"prow/cmd/plank/main.go:159","func":"main.main","level":"error","msg":"Error syncing.","time":"2019-08-16T19:51:01Z"}
{"component":"plank","duration":"8m30.189265215s","file":"prow/cmd/plank/main.go:161","func":"main.main","level":"info","msg":"Synced","time":"2019-08-16T19:51:01Z"}
```

**Comment by fejta** [comment]
Can we revert this so we can bump prow today/monday?

**Pull request merged**